### ISSUE TITLE
CSS is off a bit

### ISSUE THREAD

**Issue by OldManUmby** [tshenry/discourse-blog-post-styling#1]
The gray background is slightly off on desktop and mobile. I tried in several browsers; same results. Also, the reply button seems off too, wouldn't it look better far right or left?

See attached screenie.

Suggestions?

<img width="1173" alt="Screen Shot 2022-02-05 at 2 52 51 PM" src="https://user-images.githubusercontent.com/22547499/152658694-a5d98205-ca2e-42e9-8e08-d62c443fdbfb.png">

**Comment by seyuf** [comment]
Yep, im experiencing this as well, when using https://github.com/discourse/material-design-stock-theme 
The error does not occur on the default theme.
@OldManUmby don't hesitate to share, if you've found a fix, or eventually, a custom theme component that fixes it? :smile:
